Description: Fix for autotools not finding lzma
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 davinci (2.17-1) unstable; urgency=medium
 .
   * Initial release
Author: Davinci Developers <davinci-dev@mars.asu.edu>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- davinci-2.17.orig/configure.ac
+++ davinci-2.17/configure.ac
@@ -252,6 +252,7 @@ if test "x$have_libhdf" = "xyes"; then
 	AC_DEFINE(H5_USE_16_API, 1, [use HDF5 1.6 API])
 fi
 
+AC_CHECK_LIB(lzma,lzma_code)
 
 
 AC_CHECK_LIB([curl],[curl_easy_init])
